alicloud/service_alicloud_kvstore.go

```diff
@@ -0,0 +1,50 @@
+package alicloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/r-kvstore"
+	"github.com/denverdino/aliyungo/common"
+)
+
+func (client *AliyunClient) DescribeRKVInstanceById(id string) (instance *r_kvstore.DBInstanceAttribute, err error) {
+	request := r_kvstore.CreateDescribeInstanceAttributeRequest()
+	request.InstanceId = id
+	resp, err := client.rkvconn.DescribeInstanceAttribute(request)
+	if err != nil {
+		return nil, err
```

Comment:
I think line 16 should the following code:
```shell
if IsExceptedError(err, InvalidKVStoreInstanceIdNotFound) {
      return nil, GetNotFoundErrorFromString(fmt.Sprintf("DB instance %s is not found.", id))
}
```
And then, all of places where invoking method  DescribeRKVInstanceById can be modified following:
```
if NotFoundError(err) {
    ...
}
```

Reply:
done